common/core/gpudevice.cpp

```diff
@@ -313,6 +315,8 @@ void GpuDevice::DisplayManager::HandleRoutine() {
       HotPlugEventHandler();
     }
   }
+
+  Resume();
```

Comment:
Instead of this u can do something like this: https://github.com/01org/IA-Hardware-Composer/blob/master/common/display/pageflipeventhandler.cpp#L115 so u dont have to read/write fds unnecessarily